lix/packages/client/src/github/forkStatus.ts

```diff
@@ -0,0 +1,141 @@
+import isoGit from "../../vendored/isomorphic-git/index.js"
+import { makeHttpClient } from "../git-http/client.js"
+import type { RepoContext, RepoState } from "../openRepository.js"
+import { getMeta } from "../github/getMeta.js"
+
+export async function forkStatus(ctx: RepoContext, state: RepoState) {
+	const {
+		gitUrl,
+		debug,
+		dir,
+		cache,
+		owner,
+		repoName,
+		githubClient,
+		// setErrors,
+		gitProxyUrl,
+	} = ctx
+
+	if (!gitUrl) {
+		throw new Error("Could not find repo url, only github supported for forkStatus at the moment")
+	}
+
+	// const repo = await this
```

Comment:
guess this is a leftover of the open repo beeing async - we can remove this comment